src/mpid/ch4/src/ch4_proc.h

```diff
@@ -34,9 +34,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_av_is_local(MPIDI_av_entry_t * av)
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_AV_IS_LOCAL);
 
 #ifdef MPIDI_CH4_DIRECT_NETMOD
-    /* Ask the netmod for locality information. If it decided not to build it,
-     * it will call back up to the MPIDIU function to get the infomration. */
-    ret = MPIDI_NM_av_is_local(av);
```

Comment:
Yeah, I don't think this feature ever got fleshed out. The idea was that the netmod would hold all locality information in certain cases, for example "instant on", where normal PMI address exchange was bypassed. I agree this can probably go for now.

Reply:
I see. I think the "instant on" scenario should be implemented as an PMI bypass rather than as a netmod feature. If we setup world information including lower layer addresses in a local file, then we can replace all the current PMI exchanges with local file loading -- essentially a PMI feature rather than a netmod feature.